scraper.rb

```diff
@@ -1,3 +1,55 @@
-require File.join(File.dirname(__FILE__), "planningalerts_xml_feed", "planningalerts_xml_feed")
+require 'scraperwiki'
+require 'mechanize'
+
+agent = Mechanize.new
+
+def extract_from_p(contents, name)
+	contents.at('p:contains("'+name+'")').inner_text.split(name).last.strip
+end
+
+def scrape_page(page)
+	contents = page.search('div.span6')
+	record = {
+		"info_url" => page.uri.to_s,
+		"council_reference" => contents.search('h1').inner_text,
```

Comment:
Council reference needs whitespace around it stripped:

![selection_001](https://cloud.githubusercontent.com/assets/48945/14872009/4ddfe414-0d2b-11e6-9a91-90cf7e6e1843.png)